packages/contracts-por/contracts/TokenControllerV3.sol

```diff
@@ -626,7 +626,7 @@ contract TokenControllerV3 {
      * @param burner address of the token that can burn
      * @param canBurn true if account is allowed to burn, false otherwise
      */
-    function setCanBurn(address burner, bool canBurn) external onlyRegistryAdmin {
+    function onlyRegistryAdminOrOwner(address burner, bool canBurn) external onlyRegistryAdmin {
```

Comment:
Accidental edit?
```suggestion
    function setCanBurn(address burner, bool canBurn) external onlyRegistryAdmin {
```

Reply:
It was, code didn't even compile. Fixed that, thanks!